website/docs/releases/2024/v2024.8.md

```diff
@@ -66,22 +73,36 @@ To try out the release candidate, replace your Docker image tag with the latest
 
 -   **Source property mappings for SCIM, OAuth, SAML and Plex sources**
 
+    All source types now support Property mappings to customize how authentik should interpret the data the source provides. In addition to that, it is also now possible to sync groups and group membership from sources that provide group information. See [Property Mappings](../../sources/property-mappings/index.md).
+
 -   **RADIUS provider custom attribute support**
 
     With 2024.8 it is possible to define custom attributes for the RADIUS provider, for example vendor specific attributes like Cisco's `AV-Pair` attribute. These attributes are defined in property mappings which means they can be dynamically defined based on the user authenticating. See [RADIUS Provider](../../providers/radius/index.mdx#radius-attributes)
 
--   **SAML Source encryption support**
+-   **SAML encryption support**
+
+    It is now possible to configure a SAML Source/Provider to decrypt and validate encrypted assertions. This can be configured by certain a [Certificate-keypair](../../core/certificates.md) and selecting it in the SAML Source/Provider.
 
-    It is now possible to configure a SAML Source to decrypt and validate encrypted assertions. This can be configured by certaing a [Certificate-keypair](../../core/certificates.md) and selecting it in the SAML Source.
+-   **GeoIP Policy**
 
--   **Removal of LDAP Provider search group**
+    With the new [GeoIP Policy](../../policies/index.md#geoip-policy) it is possible to grant/deny access based on Country and ASN, without having to write an expression.
+
+-   **Simplification of LDAP Provider permissions**
 
     The LDAP provider now uses RBAC to assign the permission to search the full directory instead of requiring a dedicated group to be created. As part of the upgrade, existing search groups' users are migrated to grant the required permission to search the full directory.
```

Comment:
```suggestion
    The LDAP provider now uses RBAC to assign the permission to search the full directory instead of requiring a dedicated group to be created. As part of the upgrade, existing search groups' users are migrated into the RBAC-managed group in order to grant the required permission to search the full directory.
```

Reply:
@Jens I am not sure about my rewording. During th eupgrade, are the users in legacy, existing search groups migrated into a new group...? Or how exactly do each of those users get granted the correct permissions to search? Maybe they get assigned a role...? Needs a little more detail here.

Reply:
The users in the group are directly assigned the permission via user-bound object-specific permissions, no new group or role is created